py-polars/tests/unit/test_lazy.py

```diff
@@ -1098,11 +1098,9 @@ def test_self_join() -> None:
     out = (
         ldf.join(other=ldf, left_on="manager_id", right_on="employee_id", how="left")
         .select(
-            exprs=[
```

Comment:
Can we still pass a list as argument?

Reply:
Yes we can! There was no need to remove the list syntax here - it still works (I restored it).

The first argument is given special treatment. Code is here:
https://github.com/pola-rs/polars/blob/b5b31d165a5be99488db38d6b0a872baace0026a/py-polars/polars/utils/_parse_expr_input.py#L43-L64